### ISSUE TITLE
[Feature Request] Would a flatpak package distribution be possible?  

### ISSUE THREAD

**Issue by igorlogius** [dynobo/normcap#147]
Would it maybe be possible to add a flatpak based distribution?

I asking because i can see myself and others  running this  software on a day to day basis and 
I came to like flatpak packages since i find the concepts  it imploys  
(sandbox, dbus(gates), shared repos (ostree),  xdg desktop integration, configurable repository, ... )  rather compeling and in comparison to other alternatives complete (mature). 

Since there is already an appimage,  i guess flatpak might already be known, but just in case here are addional links: 
-  General:  https://flatpak.org/
-  Getting Started Guide: https://docs.flatpak.org/en/latest/first-build.html

Thanks for reading and maybe considering it.

**Comment by dynobo** [comment]
Thank you for your feedback, @igorlogius.

I don't have any experience with flatpak and getting into it is not very high prio for me. Still I agree, it would be nice to have!

It would be great, if someone could help out here :blush:

**Comment by Tim453** [comment]
I created a flatpak for NormCap. You can find the manifest [ here ](https://github.com/Tim453/com.github.dynobo.normcap). @dynobo Do you want me to publish your application on Flathub?

**Comment by dynobo** [comment]
Hey @Tim453 , that's awesome!! How about we integrate building (and publishing?) the flatpack as a Github action right into the CI/CD of the NormCap repo? Would you be willing to provide a PR? :-)

**Comment by Tim453** [comment]
I think it would be better if we let Flathub build and publish the Flatpak. So people can find it more easily in there [Store](https://flathub.org/home) and they will also receive automatic updates. Therefore i can create a repository in the [Flathub Github project](https://github.com/flathub) which contains the manifest to build the application.

**Comment by dynobo** [comment]
@Tim453 , ahh, good to know, I haven't looked into flatpack hub before. I'll take it up from here and create a Flathub account for NormCap. Thanks a lot for looking into this and providing the manifest!!

**Comment by dynobo** [comment]
Create [PR#3352](https://github.com/flathub/flathub/pull/3352) at FlatHub.

**Comment by pizzadude** [comment]
@dynobo Hi, I found this program on flathub. The flatpak doesn't work for me in KDE wayland, when capturing I get an error "22:36:44 - ERROR   - normcap.gui.utils:153 - Invalid dbus interface". I use KDE version 5.25.3 on Fedora Linux if that helps.

The AppImage works fine for me in KDE wayland though.

**Comment by dynobo** [comment]
@pizzadude , thanks a lot for reporting this! I'm pretty new to flatpak, so it might need 1-2 more tries for a stable package... 
I'm looking into it and track this as a separate issue:

**Comment by igorlogius** [comment]
Nice. 

```
flatpak info com.github.dynobo.normcap

NormCap - Extract text from any image

          ID: com.github.dynobo.normcap
         Ref: app/com.github.dynobo.normcap/x86_64/stable
        Arch: x86_64
      Branch: stable
     Version: 0.3.4
     License: GPL-3.0-or-later
      Origin: flathub
  Collection: org.flathub.Stable
Installation: user
   Installed: 629,2 MB
     Runtime: org.kde.Platform/x86_64/6.2
         Sdk: org.kde.Sdk/x86_64/6.2

      Commit: 528538e27dbd41c9f3f6c7a8598476c70e45fa80c911ca00329c5205cbde02c9
     Subject: Use x11 as fallback (80840363)
        Date: 2022-07-29 06:30:39 +0000
```

Thanks  @Tim453 and @dynobo for looking into this.

**Comment by dynobo** [comment]
Closing this issue, as the flatpak (for v0.3.4) is now available on:
https://flathub.org/apps/details/com.github.dynobo.normcap

Follow ups:
Still there are some issues to fix (#224), and I have to somehow integrate that into automated deployment (#225). And the information page on flathub could be more informative (#226).
